src/View/Helper/FormHelper.php

```diff
@@ -1008,7 +1008,8 @@ public function input($fieldName, array $options = [])
             'required' => null,
             'options' => null,
             'templates' => [],
-            'templateVars' => []
+            'templateVars' => [],
+            'optionsLabel' => true
```

Comment:
;-) I wanted to differentiate from the input label, and relates more to the options.